components/authz-service/storage/v2/postgres/postgres_test.go

```diff
@@ -4410,7 +4410,9 @@ func TestCreateProject(t *testing.T) {
 			}
 			resp, err := store.CreateProject(ctx, &oneProjectTooMany)
 			assert.Nil(t, resp)
-			assert.Equal(t, storage_errors.ErrMaxProjectsExceeded, err)
+			assert.Error(t, err)
```

Comment:
could we add a test for configuring a different value for the project limit?

Reply:
we can't without refactoring the whole test suite :D but i'll do it for you if you want